src/sentry/migrations/0429_backfill_substatus_for_groupstatus_ignored.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 2.2.28 on 2023-04-25 22:40
+
+from django.db import connection, migrations
+from psycopg2.extras import execute_values
+
+from sentry.models import GroupStatus, GroupSubStatus
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapper
+
+BATCH_SIZE = 100
+
+UPDATE_QUERY = """
+    UPDATE sentry_groupedmessage
+    SET substatus = %s
+    FROM (VALUES %s) as data (id)
+    WHERE sentry_groupedmessage.id = data.id
```

Comment:
Might also be safer to include `id, status, substatus` in the tuple and add an extra where clause `sentry_groupedmessage.id = data.id and sentry_groupedmessage.status = data.status` to avoid mutating rows that have since been updated by the user.